service/controller/key/cluster.go

```diff
@@ -115,6 +115,17 @@ func MasterInstanceType(cluster v1alpha1.Cluster) string {
 	return clusterProviderSpec(cluster).Provider.Master.InstanceType
 }
 
+func ManagedRecordSets(cluster v1alpha1.Cluster) []string {
+	tcBaseDomain := TenantClusterBaseDomain(cluster)
+	return []string{
+		fmt.Sprintf("%s.", tcBaseDomain),
+		fmt.Sprintf("\\052.%s.", tcBaseDomain), // \\052 - `*` wildcard record
```

Comment:
Is wildcard record created by external-dns?

Reply:
Nope, wildcard created by aws-operator. It is the same as in azure-operator. external-dns doesn't support multiple records per services/ingress

Reply:
Won't it be deleted by the CF stack then?

Reply:
I meant it is created by CF stack in aws-operator. So it will be deleted by CF stack.

Reply:
Then it shouldn't be listed here correct?

Reply:
The list here is the list of managed by operator records. Not by apps. So cleanup delete all the others which are not in list.

Reply:
Ack. That makes sense. Thanks for bearing with me.